typescript/cli/src/config/chain.ts

```diff
@@ -111,6 +114,42 @@ export async function createChainConfig({
   }
 }
 
+async function addBlockExplorerConfig(metadata: ChainMetadata): Promise<void> {
+  const wantBlockExplorerConfig = await confirm({
+    default: false,
+    message: 'Do you want to add a block explorer config for this chain',
+  });
+  if (wantBlockExplorerConfig) {
+    const name = await input({
+      message: 'Enter a human readable name for the explorer:',
+    });
+    const url = await input({
+      message: 'Enter the base URL for the explorer:',
+    });
+    const apiUrl = await input({
+      message: 'Enter the base URL for requests to the explorer API:',
+    });
+    const family = (await select({
+      message: 'Select the type (family) of block explorer:',
+      choices: Object.entries(ExplorerFamily).map(([_, value]) => ({ value })),
+      pageSize: 10,
+    })) as ExplorerFamily;
+    const apiKey =
+      (await input({
+        message:
+          "Optional: Enter an API key for the explorer, or 'enter' to skip. Please be sure to remove this field if you intend to add your config to the Hyperlane registry:",
```

Comment:
just to disambiguate "enters"
```suggestion
          "Optional: Provide an API key for the explorer, or hit 'enter' to skip. Please be sure to remove this field if you intend to add your config to the Hyperlane registry:",
```

Reply:
done